cmd/rekor-server/app/serve.go

```diff
@@ -103,11 +103,15 @@ var serveCmd = &cobra.Command{
 		server.Port = int(viper.GetUint("port"))
 		server.EnabledListeners = []string{"http"}
 
-		// Update logRangeMap if flag was passed in
+		// Build logRangeMap if related flags were passed in
 		rangeMap := viper.GetString("trillian_log_server.log_id_ranges")
+		tlogID := viper.GetInt64("trillian_log_server.tlog_id")
 		if rangeMap != "" {
-			if err := logRangeMap.Set(rangeMap); err != nil {
-				log.Logger.Fatal("unable to set logRangeMap from flag: %v", err)
+			if tlogID == 0 {
```

Comment:
we don't need this check because we check for it later in `logRangeMap.Set` again